i18n/locales/de/common.json

```
@@ -1,238 +1,532 @@
+  "Oops! Wrong passphrase": "Hoppla! Falsche Passphrase",
+  "Oops, looks like something went wrong. Please try again.": "Ups, sieht so aus, als wäre etwas schief gelaufen. Bitte versuche es erneut. ",
+  "Outgoing transactions": "Ausgehende Transaktionen",
+  "Paper wallet": "Papier wallet",
   "Passphrase": "Passphrase",
-  "Passphrase contains extra whitespace between words": "Passphrase enthält zusätzliche Leerzeichen zwischen den Wörtern",
-  "Passphrase contains unnecessary whitespace at the beginning": "Die Passphrase enthält ein unnötiges Leerzeichen am Anfang",
-  "Passphrase contains unnecessary whitespace at the end": "Die Passphrase enthält unnötige Leerzeichen am Ende",
-  "Passphrase is not valid": "Die Passphrase ist nicht gültig",
-  "Passphrase of the account is saved till the end of the session.": "Die Passphrase für das Konto wird bis zum Ende der Sitzung gespeichert.",
-  "Passphrase of the account will be required to perform any transaction.": "Die Passphrase für das Konto wird benötigt um Transaktionen jeglicher Art durchzuführen.",
-  "Passphrase should have 12 words, entered passphrase has {{length}}": "Die Passphrase sollte 12 Wörter haben, die eingegebene Passphrase hat {{length}}",
-  "Password": "Passwort",
+  "Passphrase is not valid": "Passphrase ist nicht gültig",
+  "Passphrase should have 12 words, entered passphrase has {{length}}": "Die Passphrase sollte 12 Wörter lang sein, die eingegebene Passphrase hat {{Length}}.",
   "Paste": "Einfügen",
-  "Peer": "Peer",
-  "Please click Next, then move around your mouse randomly to generate a random passphrase.": "Bitte klicke auf \"Weiter\" und bewege dann deine Maus willkürlich über den Bildschirm, um eine zufällige Passphrase zu erzeugen.",
-  "Please keep it safe!": "Bitte sicher aufbewahren!",
-  "Press #{key} to copy": "Zum Kopieren #{key} drücken",
-  "Processing delegate names: ": "Verarbeite Delegierte Namen:",
-  "Proxy Authentication": "Proxy Authentifizierung",
-  "Public Key": "Public Key",
-  "Public key : ": "Public Key:",
+  "Pending...": "Ausstehend.....",
+  "Percentage of successfully forged blocks of when the delegate should have forged a block of transactions.": "Prozentsatz der erfolgreich forged Blöcke, in dem der Delegierte einen Block von Transaktionen forged haben sollte.",
+  "Perhaps your question has already been raised in our FAQ": "Vielleicht wurde deine Frage bereits in unserer (HGF) Häufig gestellte Fragen gestellt.",
+  "Personalize each transaction with a custom message. Look up its value in a fiat currency of your choice.": "Personalisiere jede Transaktion mit einer benutzerdefinierten Nachricht. Schau dir den Wert in einer Fiat-Währung deiner Wahl an.",
+  "Please check the address": "Bitte überprüft die Adresse",
+  "Please check the highlighted words": "Bitte überprüfe die markierten Wörter.",
+  "Please select the account you’d like to sign in to or": "Bitte wählt das Konto aus, bei dem ihr euch anmelden wollt oder",
```

One more formal/informal you:
```suggestion
  "Please select the account you’d like to sign in to or": "Bitte wählt das Konto aus, bei dem du dich anmelden wollst oder",
```